k8s/upgrades/0.9.0-1.0.0/label_patch.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+ns=$1
```

Comment:
I think we need to set ns to `openebs` based on the arguments passed to script.

Reply:
this script is taking the namespace in which openebs is installed as input